Objects/dictobject.c

```diff
@@ -2080,8 +2089,8 @@ PyDict_Clear(PyObject *op)
         dictkeys_decref(interp, oldkeys);
     }
     else {
-       assert(oldkeys->dk_refcnt == 1);
-       dictkeys_decref(interp, oldkeys);
+        assert(oldkeys->dk_refcnt == 1 || oldkeys == Py_EMPTY_KEYS);
+        dictkeys_decref(interp, oldkeys);
```

Comment:
Nice catch about wrong indent.
I think no need to update the assert, because we have this code above:

```c
    if (oldkeys == Py_EMPTY_KEYS) {
        return;
    }
```

Reply:
fixed